nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/src/main/java/org/apache/nifi/processors/kite/AbstractKiteProcessor.java

```diff
@@ -44,8 +44,31 @@
 import org.kitesdk.data.URIBuilder;
 import org.kitesdk.data.spi.DefaultConfiguration;
 
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.nifi.hadoop.KerberosProperties;
+import org.apache.nifi.hadoop.SecurityUtil;
+import org.apache.nifi.processor.ProcessorInitializationContext;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.security.PrivilegedExceptionAction;
+import java.util.concurrent.atomic.AtomicReference;
+
 abstract class AbstractKiteProcessor extends AbstractProcessor {
 
+    protected KerberosProperties getKerberosProperties(File kerberosConfigFile) {
```

Comment:
Ah that makes more sense. Are you able to refactor the PR without the NIFI-385 changes?